packages/@okta/idx-js/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "idx-for-signin-widget",
+  "version": "1.0.0",
+  "description": "copies dist folder of idx-js into idx-dist for consumption in widget",
+  "scripts": {
+    "build": "yarn --cwd packages/@okta/idx-js &&  cp -R node_modules/@okta/okta-idx-js/dist/. ../idx-dist/"
```

Comment:
Aren't we already in this directory? I think the --cwd is unnecessary

Reply:
fixed